examples/counter-next/web-remix/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+export RUSTFLAGS='--cfg getrandom_backend="wasm_js"'
+
+pushd ../shared && RUST_LOG=info wasm-pack build --target web
```

Comment:
Instead of this, In my PR I've avoided `rand` in wasm and used `js_sys` for random numbers - get them directly from the browser. 

Might be a better way, and avoids needing various shell scripts and magic env vars...?